server/swagger.json

```diff
@@ -3999,6 +3999,13 @@
 									"$ref": "#/definitions/DashboardQuery"
 								}
 							},
+              "axes": {
+                "description": "The viewport for a cell's visualizations",
+                "type": "object",
+                "additionalProperties": {
```

Comment:
Yes, changing the docs are good.  However, that reminds me.  You'll need to check for only x, y, y2 when the data comes into the server.